routes/index.js

```diff
@@ -19,6 +19,19 @@ router.get('/rules', function(req, res) {
 });
 
 router.post('/invite', function(req, res) {
+
+  var checkEmail = require('legit');
+  // Checks for the existence of a MX record for the email address' domain
+  checkEmail(req.body.email, function(validation, addresses, err) {
+    if (!validation) {
+      res.render('result', {
+        community: config.community,
+        message: 'Failed. Email domain is invalid.',
+        isFailed: true
+      });
+    }
```

Comment:
Did you validate this works as expected? I think we may need to force a return here like this: https://github.com/lorenanicole/slack-invite-automation/blob/master/routes/index.js#L63.